### PR TITLE
Update doc on dotfiles

### LINKED ISSUE #179: Doc on dotfiles needs update
[Doc Link](https://docs.microsoft.com/en-us/visualstudio/online/reference/personalizing)

In the current doc it shows the `dotfiles command` as being able to run from anywhere.

image
It seems that it's since been changed to run from inside the cloned dotfiles repo

### PR DESCRIPTION
**Pull request by jfversluis**
Takes into account the latest developments in this area as well as the remarks in #179 

Fixes #179

### CODE REVIEW

**Comment by jfversluis** [comment]
cc: @abuchholtzau

File: vs-online/reference/personalizing.md [hunk 1]
```diff
@@ -27,24 +27,38 @@ Press `F1` and select the **Preferences: Open Setting (UI)** command in the [com
...
 
 In both experiences, there are three options that can be configured:
 
 1. **Dotfiles Repository**: The URL of the Git repository containing your dotfiles. (*Required to personalize an environment, optional otherwise*)
-2. **Dotfiles Target Path**: The path where the dotfiles repo will be cloned. Defaults to `~/dotfiles`. (*Optional*)
+2. **Dotfiles Target Path**: The path where the dotfiles repo will be cloned. Defaults to `~/dotfiles` on Linux, `C:\dotfiles` on Windows. (*Optional*)
 
-3. **Dotfiles Install Command**: The command to run after cloning the dotfiles repository. By default, Codespaces scans the dotfiles repository and runs one of the following files:
+3. **Dotfiles Install Command**: The command to run after cloning the dotfiles repository. The install command should be relative to the repository root folder.
+By default, Codespaces scans the dotfiles repository and runs one of the following files. On a Linux-based Codespace:
```
Comment (abuchholtzau) [inline vs-online/reference/personalizing.md:40]:
```suggestion
By default, Codespaces scans the dotfiles repository and runs one of the following files. 

On Linux-based codespaces:
```

File: vs-online/reference/personalizing.md [hunk 2]
```diff
@@ -27,24 +27,38 @@ Press `F1` and select the **Preferences: Open Setting (UI)** command in the [com
...
     - `install.sh`
     - `install`
     - `bootstrap.sh`
     - `bootstrap`
     - `setup.sh`
     - `setup`
+    
+    On a Windows-based Codespace:
```
Comment (abuchholtzau) [inline vs-online/reference/personalizing.md:48]:
```suggestion
    On Windows-based codespaces:
```

File: vs-online/reference/personalizing.md [hunk 3]
```diff
@@ -27,24 +27,38 @@ Press `F1` and select the **Preferences: Open Setting (UI)** command in the [com
...
+    - `bootstrap.ps1`
+    - `bootstrap.bat`
 
-If none of these files are found, any files and folders starting with `.` are symlinked to the home (`~` or `$HOME` on Linux) directory.
+If none of these files are found, any files and folders starting with `.` are symlinked to the home (`~` or `$HOME` on Linux, `%HOMEPATH%` on Windows) directory. In addition, a couple of files are picked up automatically by convention for sourcing.
+
+On a Linux-based Codespace:
```
Comment (abuchholtzau) [inline vs-online/reference/personalizing.md:54]:
```suggestion
On Linux-based codespaces:
```

File: vs-online/reference/personalizing.md [hunk 4]
```diff
@@ -27,24 +27,38 @@ Press `F1` and select the **Preferences: Open Setting (UI)** command in the [com
...
+  - `sourcer.zsh` for zsh shell
+  - `sourcer.sh` for Bash and other shell
+
+On a Windows-based Codespace:
```
Comment (abuchholtzau) [inline vs-online/reference/personalizing.md:58]:
```suggestion
On Windows-based codespaces:
```

**Comment by abuchholtzau** [comment]
Suggested a few edits; mainly converting to a lowercase "codespace" when we're talking about a particular instance.

**Comment by abuchholtzau** [comment]
> cc: @abuchholtzau
> 
> I also noticed that this is all specific to VS Code while this is now also available in the web portal as well as Visual Studio 2019. Would it be enough to just include a note about that here? Or should we spilt that out in different pages?

@gregvanl thoughts here? Do you have suggestion on how we can convey this works across all clients? The dotfiles apply to the environment but I agree that it seems to implicitly indicate that it's a VS Code feature based on our current docs.

**Pull request merged**